app/src/main/java/com/ncnf/database/builder/UserBuilder.java

```diff
@@ -44,7 +45,7 @@ public User toObject(String uuid, Map<String, Object> data) {
             Date birthDate = ((Timestamp) data.get(BIRTH_DATE_KEY)).toDate();
             boolean notifications = (boolean) data.get(NOTIFICATIONS_KEY);
 
-            return new User(db, uuid, username, email, firstName, lastName, friends, ownedGroups, participatingGroups, savedEvents, notifications, birthDate);
+            return new User(db, uuid, username, email, firstName, lastName, friends, ownedGroups, participatingGroups, savedEvents, notifications, birthDate, null);
```

Comment:
Will this require a change in the database? I'm collecting all the things we would have to modify in it this week

Reply:
The database will change but it'll be done automatically, as of right now there are already some users with this field and it's working as expected :)

Reply:
Oh cool!